src/resources/postcss/views/month/_multiday-event.pcss

```diff
@@ -5,10 +5,34 @@
 	 * ----------------------------------------------------------------------------- */
 
 	.tribe-events-calendar-month__event-multiday {
-		border-radius: 6px;
-		margin-bottom: var(--spacer-0);
-		background-color: white;
-		overflow: hidden;
+		display: none;
+
+		@media (--viewport-medium) {
+			display: block;
+			border-radius: 6px;
+			background-color: white;
+			overflow: hidden;
+			position: absolute;
+			z-index: calc( var(--z-index-spinner-container) - 1 );
+		}
+	}
+
+	.tribe-events-calendar-month__event-multiday--start {
+		@media (--viewport-medium) {
+			border-radius: 6px 0 0 6px;
+			&.tribe-events-calendar-month__event-multiday--end {
```

Comment:
instead of repeating this style here and below with `&.tribe-events-calendar-month__event-multiday--start`, we can do one of two things:

1. we can target `border-top-left-radius` and `border-bottom-left-radius` for `.tribe-events-calendar-month__event-multiday--start`, and `border-top-right-radius` and `border-bottom-right-radius` for `.tribe-events-calendar-month__event-multiday--end`
2. add a style for `.tribe-events-calendar-month__event-multiday--start.tribe-events-calendar-month__event-multiday--end` below both these styles with a `border-radius: 6px`.

I'd go with approach 1 for modularity.